convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -47,13 +46,59 @@ def __call__(self, *args, **kwargs):
             self.used_args.append(args)
             return self.output_tuple
 
+    class PathExistsMocked(unit_tests.MockFunction):
+        def __init__(self, return_value=True):
+            self.return_value = return_value
+
+        def __call__(self, filepath):
+            return self.return_value
+
     class GenerateRpmVaMocked(unit_tests.MockFunction):
         def __init__(self):
             self.called = 0
 
         def __call__(self):
             self.called += 1
 
+    class GetLoggerMocked(unit_tests.MockFunction):
+        def __init__(self):
+            self.task_msgs = []
+            self.info_msgs = []
+            self.warning_msgs = []
+            self.critical_msgs = []
+
+        def __call__(self, msg):
+            return self
+
+        def critical(self, msg):
+            self.critical_msgs.append(msg)
+            raise SystemExit(1)
+
+        def task(self, msg):
+            self.task_msgs.append(msg)
+
+        def info(self, msg):
+            self.info_msgs.append(msg)
+
+        def warn(self, msg, *args):
+            self.warning_msgs.append(msg)
+
+        def warning(self, msg, *args):
+            self.warn(msg, *args)
+
+        def debug(self, msg):
+            pass
+
+    class GetFileContentMocked(unit_tests.MockFunction):
+        def __init__(self, data):
+            self.data = data
+            self.as_list = True
+            self.called = -1
+
+        def __call__(self, filename, as_list):
+            self.called += 1
+            return self.data[self.called]
```

Comment:
```suggestion
            self.called = 0

        def __call__(self, filename, as_list):
            self.called += 1
            return self.data[self.called + 1]
```

Reply:
This will raise: `IndexError: list index out of range` since list start in 0 and in test will add only one item..

Reply:
That's why I suggest `self.data[self.called - 1]`. It does not make sense for `called` to be off by one